src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs/launch_analysis.cljs

```diff
@@ -136,11 +135,12 @@
 (react/defc LaunchAnalysisButton
   {:render
    (fn [{:keys [props]}]
-     [comps/Button {:text "Launch Analysis..."
-                    :disabled? (:disabled? props)
-                    :onClick #(if (:force-login? props)
-                               (sign-in/show-sign-in-dialog :refresh-token (:after-login props))
-                               (modal/push-modal [Form (select-keys props [:config-id :workspace-id :root-entity-type :on-success :cromwell-version])]))}])})
+     [comps/Button
+      {:text "Launch Analysis..."
+       :disabled? (:disabled? props)
+       :onClick (modal/push-modal
+                 [Form (select-keys props [:config-id :workspace-id
+                                           :root-entity-type :on-success :cromwell-version])])}])})
```

Comment:
why was the :force-login option removed here?

My understanding is this option handled the (corner) case where the user's refresh token expired, based on our definition of expire, within the user's current session. Before launching an analysis, we check the refresh token to see if it's still valid, and if not, we go get a new one. In this PR, we only check the refresh token date upon login, correct?